h/cli/commands/publisher.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+import click
+
+from h import models
+
+
+@click.group()
+def publisher():
+    """Manage publisher accounts."""
+
+
+@publisher.command()
+@click.option('--name', prompt=True, help="The name of the publisher")
+@click.option('--authority', prompt=True, help="The authority (domain name) for the publisher")
+@click.pass_context
+def add(ctx, name, authority):
+    """
+    Create a new publisher account.
+
+    This consists of an authority, an admin user and the main group for the
+    publisher's annotations.
+    """
+    request = ctx.obj['bootstrap']()
+
+    # Create a new authority.
+    authclient = models.AuthClient(name=name, authority=authority)
+    request.db.add(authclient)
+
+    # Add an admin user for the authority.
+    signup_service = request.find_service(name='user_signup')
+    creator = signup_service.signup(username=u'admin',
+                                    email=u'admin@localhost',
+                                    authority=authority,
+                                    require_activation=False)
+
+    # Add a main group for the publisher's annotations.
+    group_svc = request.find_service(name='group')
+    group_svc.create(name=name, authority=authority, userid=creator.userid,
+                     type_='publisher')
+
+    id = authclient.id
+    secret = authclient.secret
+
+    request.tm.commit()
+
+    click.echo('Publisher account for {authority} created'.format(authority=authority), err=True)
```

Comment:
This was just blindly adapted from the `h/cli/commands/user.py`. I suppose this is because the output was just informational in that context. I'll switch to stdout here.